services/backend/src/migrator/migrator.go

```diff
@@ -29,7 +29,7 @@ func NewMigrator() *Migrator {
 		}
 	}
 	return &Migrator{
-		db: (*dbConn).DB.DB,
+		db: (*dbConn).DB,
```

Comment:
The reason why I pass in an sql instead of an sqlx struct is because golang migrator can only use sql structs so I have no benefit of using sqlx. In case I want to switch to something else, I just have to change it at one point and leave the migrator as is